make/go/dep_protoc_gen_connect_go.mk

```diff
@@ -7,16 +7,16 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://github.com/bufbuild/connect-go 20230627 checked 20230703
-CONNECT_VERSION ?= v1.9.0
+# https://github.com/connectrpc/connect-go 20230726 checked 20230816
+CONNECT_VERSION ?= v1.11.0
 
 GO_GET_PKGS := $(GO_GET_PKGS) \
-	github.com/bufbuild/connect-go@$(CONNECT_VERSION)
+	github.com/connectrpc/connect-go@$(CONNECT_VERSION)
 
 PROTOC_GEN_CONNECT_GO := $(CACHE_VERSIONS)/connect-go/$(CONNECT_VERSION)
 $(PROTOC_GEN_CONNECT_GO):
 	@rm -f $(CACHE_BIN)/connect-go
```

Comment:
I know this isn't new but I wonder if we can remove this now - I don't think there is a `connect-go` binary?

Reply:
This should be `protoc-gen-connect-go`